tests/expressions/tests.py

```diff
@@ -1094,3 +1094,33 @@ def test_aggregates(self):
         self.assertEqual(repr(StdDev('a')), "StdDev(F(a), sample=False)")
         self.assertEqual(repr(Sum('a')), "Sum(F(a))")
         self.assertEqual(repr(Variance('a', sample=True)), "Variance(F(a), sample=True)")
+
+
+class RefreshOnSaveTests(TestCase):
+    def test_insert(self):
+        empl = Employee(
```

Comment:
Use a single line (we allow up to 119 characters when it helps readability) and `.objects.create()` rather than `.save()`